### PR TITLE
Added: sshpass

### PR DESCRIPTION
**Pull request by Toasterson**
A little small utility to allow login via password in file/environement variable or prompt. Tested and Works

### CODE REVIEW

File: components/network/sshpass/Makefile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+build:		$(BUILD_32)
+
+install:	$(INSTALL_32)
+
+test:		$(TEST_32)
```
Comment (pyhalov) [inline components/network/sshpass/Makefile:38]: Does it have any build dependencies?

File: components/network/sshpass/Makefile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=sshpass
+COMPONENT_VERSION=1.06
+COMPONENT_SUMMARY=Non-interactive ssh password auth
+COMPONENT_PROJECT_URL=https://sourceforge.net/projects/sshpass
+COMPONENT_FMRI=network/$(COMPONENT_NAME)
+COMPONENT_CLASSIFICATION=
```
Comment (pyhalov) [inline components/network/sshpass/Makefile]: Please, fill it

File: components/network/sshpass/Makefile [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_URL=  https://sourceforge.net/projects/sshpass/files/sshpass/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_ARCHIVE_HASH= sha256:c6324fcee608b99a58f9870157dfa754837f8c48be3df0f5e2f3accf145dee60
+COMPONENT_LICENSE=	CPL
```
Comment (pyhalov) [inline components/network/sshpass/Makefile]: You set CPL here, but I read GPLv2 in COPYING

File: components/network/sshpass/Makefile (hunk 1 above)
Comment (Toasterson) [reply components/network/sshpass/Makefile:38]: gcc and make are we adding those aswell?

**Pull request merged**